frontend/tests/features/collection/revision.test.ts

```diff
@@ -137,22 +140,23 @@ describe("Collection Revision", () => {
 });
 
 async function startRevision(page: Page): Promise<string> {
-  await goToPage(TEST_URL + ROUTES.MY_COLLECTIONS, page);
```

Comment:
Lines 140-143 are moved to lines 148-150 due to a change in flow after a collection is deleted.